msteams-platform/resources/troubleshoot.md

```diff
@@ -91,14 +91,22 @@ Common reasons for manifest read errors:
 
 * Invalid JSON. Use an IDE such as [Visual Studio Code](https://code.visualstudio.com) or [Visual Studio](https://www.visualstudio.com/vs/) that automatically validates the JSON syntax.
 * Encoding issues. Use UTF-8 for the *manifest.json* file. Other encodings, specifically with the BOM, may not be readable.
-* Malformed .zip package. The *manifest.json* file must be at the top level of the .zip file. Note that default Mac file compression might place the *manifest.json* in a subdirectory, which won't properly load in Microsoft Teams.
+* Malformed .zip package. The *manifest.json* file must be at the top level of the .zip file. Default Mac file compression might place the *manifest.json* in a subdirectory, which won't properly load in Microsoft Teams.
 
 ### Another extension with same ID exists
 
 If you're attempting to upload an updated package with the same ID again, choose the **Replace** icon at the end of the tab's table row rather than the **Upload** button.
 
 If you're not re-uploading an updated package, ensure that the ID is unique.
 
+## Disable anonymous users or bot joining Teams meeting
+
+* Disable RSC permissions for all apps, which prevent apps added for new meetings without users consent. Apps that users already installed for existing meetings might still access those meetings. To remove RSC permission for existing apps in the meeting:
```

Comment:
This is not correct. There should be no mention of RSC permissions here at all.

Also, it is not clear what this line means "Disable anonymous users or bot joining Teams meeting" - are we talking about an 'anonymous bot' here? How are an anonymous user and a bot related?

These should be separate sections to say the least.

"Disable anonymous users from joining a Teams meeting" should have only steps for disabling anonymous users.

Reply:
Removed RSC related information and updated the doc as suggested.